src/singularity.ts

```diff
@@ -28,7 +28,7 @@ export const updateSingularityPenalties = (): void => {
 }
 
 function getSingularityOridnalText(singularityCount: number): string {
-  return `${i18next.t('general.youAreInThe')} <span style="color: gold">${toOrdinal(singularityCount)} Singularity</span>`
+  return `${i18next.t('general.youAreInThe')} <span style="color: gold">${toOrdinal(singularityCount)} ${i18next.t('general.singularity')}</span>`
```

Comment:
this won't be correct in all languages. We'd have to move this entire line to a singular i18n line.

Reply:
Pushed a commit to fix this!